### PR TITLE
Use GraphQL Batch for find by id fields

### PR DESCRIPTION
**Pull request by cjoudrey**
First stab at adding batching to simple find by id fields.

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 development:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: secret-for-development-not-so-secret
```
Comment (cjoudrey) [inline config/secrets.yml:2]: This was starting to annoy me. It kept asking me to set an ENV var in dev/test.

Comment (xuorig) [reply config/secrets.yml:2]: 👍

File: app/models/graph/types/query.rb [hunk 2]
```diff
@@ -8,56 +8,32 @@ module Types
       field :node, GraphQL::Relay::Node.field
       field :nodes, GraphQL::Relay::Node.plural_field
 
-      field :person, Graph::Types::Person do
-        argument :id, types.ID
-        resolve ->(_, args, _) { ::Person.find_by(id: args['id']) }
-      end
-
+      field :person, Graph.find_by_id_field(Graph::Types::Person, ::Person)
```
Comment (xuorig) [inline app/models/graph/types/query.rb:11]: nice 👌

File: app/models/graph.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+module Graph
+  class << self
+    def find_by_id_field(type, model)
+      GraphQL::Field.define do
+        type type
```
Comment (xuorig) [inline app/models/graph.rb:5]: DSL 👯‍♂️

**Pull request merged**